src/Illuminate/Database/Migrations/Migrator.php

```diff
@@ -4,13 +4,25 @@
 
 use Illuminate\Support\Arr;
 use Illuminate\Support\Str;
+use Illuminate\Events\Dispatcher;
```

Comment:
This import should be from the `Contracts` namespace as you put it in the docblock.

Reply:
Good point. Changed.